packages/vuetify/src/components/VProgressLinear/VProgressLinear.ts

```diff
@@ -42,61 +54,101 @@ export default mixins(Colorable).extend({
     },
     height: {
       type: [Number, String],
-      default: 7
+      default: 4
     },
     indeterminate: Boolean,
     query: Boolean,
+    reactive: Boolean,
+    rounded: Boolean,
+    stream: Boolean,
+    striped: Boolean,
     value: {
       type: [Number, String],
       default: 0
     }
   },
 
+  data () {
+    return {
+      internalLazyValue: this.value || 0
+    }
+  },
+
   computed: {
+    __cachedBackground (): VNode {
```

Comment:
What is the rationale and strategy behind these `__cachedX` computed that return VNodes. What determines if a VNode should be generated through this or a `genX` function? Why the `__`?

Reply:
Something that was originally done with `v-img` and I just copied the naming convention. Non-slotted generated components are more performant as computed properties.

Reply:
If this is a convention we should be using going forward, it should be documented somewhere and communicated. And `genBuffer` could be made a computed prop as well.

Reply:
Also, `__cachedBarType` should just return either determinate or indeterminate from `gen` functions. Right now both will be computed and created but only one of them is ever used at the same time.

Reply:
Computed properties are lazy so this is not applicable.